scipy/stats/stats.py

```diff
@@ -3460,15 +3460,16 @@ def pointbiserialr(x, y):
 KendalltauResult = namedtuple('KendalltauResult', ('correlation', 'pvalue'))
 
 
-def kendalltau(x, y, initial_lexsort=None, nan_policy='propagate', method='auto'):
+def kendalltau(x, y, initial_lexsort=None, nan_policy='propagate', method='auto',
+               variant='taub'):
```

Comment:
Maybe call the parameter type instead of variant, with possible values a, b, c

Reply:
I don't like the idea of overriding the built-in `type()` function of Python like this.

Reply:
I think I agree with "variant" or, as I was thinking, "version". But maybe variant is best.